staging/src/k8s.io/dynamic-resource-allocation/resourceslice/resourceslicecontroller.go

```diff
@@ -491,14 +491,17 @@ func (c *Controller) syncPool(ctx context.Context, poolName string) error {
 
 	if pool, ok := resources.Pools[poolName]; ok {
 		// Match each existing slice against the desired slices.
-		// Two slices match if they contain exactly the same
-		// device IDs, in an arbitrary order. Such a matched
-		// slice gets updated with the desired content if
-		// there is a difference.
+		// Two slices "match" if they contain exactly the
+		// same device IDs, in an arbitrary order. As a
+		// special case, slices are also considered
+		// "matched" in the scenario where there's a single
+		// existing slice and a single desired slice. Such a
+		// matched slice gets updated with the desired
+		// content if there is a difference.
 		//
-		// This supports updating the definition of devices
-		// in a slice. Adding or removing devices is done
-		// by deleting the old slice and creating a new one.
+		// In the case where there is more than one existing
+		// or desired slices, adding or removing devices is
+		// done by deleting the old slice and creating a new one.
```

Comment:
The last line could have been left unchanged by not reflowing automatically, but this is good enough.